File: CS/Form1.cs
using System;
using System.Windows.Forms;

using DevExpress.XtraReports.UI;
// ...

namespace XtraReport_PrintingCS {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            // Create a report.
            XtraReport1 report = new XtraReport1();

            // Invoke the Print dialog to print the report.
            report.PrintDialog();

            // Send the report directly to the default printer.
            report.Print();
        }

    }
    
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms

Imports DevExpress.XtraReports.UI
' ...

Namespace XtraReport_PrintingCS
	Public Partial Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a report.
			Dim report As XtraReport1 = New XtraReport1()

			' Invoke the Print dialog to print the report.
			report.PrintDialog()

			' Send the report directly to the default printer.
			report.Print()
		End Sub

	End Class

End Namespace